src/regions/ca/i18n/translation.en.ts

```diff
@@ -64,20 +79,5 @@ export default {
       ],
       ['People with travel plans', 'travel-plans']
     ]
-  },
-  provinces: {
```

Comment:
I would've kept provinces as they're used in other places, and fyi you can do references to other keys in translations 
E.g.
```
[ '$t(provinces.QC)', 'ca-qc' ]
```